cores/piduino/pgmspace.h

```diff
@@ -1,7 +0,0 @@
-#ifndef PGMSPACE_INCLUDE
```

Comment:
didn't we talk about not creating an avr folder in the core and me making it work another way?
please move pgmspace.h back in the core folder and I'll take care of the rest